graviti/dataframe/column/series.py

```diff
@@ -502,3 +542,26 @@ def __getitem__(self, key: int) -> Optional[FileBase]:
             return None
 
         return FileBase.from_pyarrow(scalar)
+
+    @overload
+    def __setitem__(self, key: slice, value: Iterable[FileBase]) -> None:
+        ...
+
+    @overload
+    def __setitem__(self, key: int, value: FileBase) -> None:
+        ...
+
+    def __setitem__(
+        self,
+        key: Union[int, slice],
+        value: Union[FileBase, Iterable[FileBase]],
+    ) -> None:
+        try:
+            if isinstance(key, int):
+                self._data.set_item(key, value.to_pyobj())  # type: ignore[union-attr]
+            else:
+                self._data.set_slice_iterable(
+                    key, (item.to_pyobj() for item in value)  # type: ignore[union-attr]
+                )
+        except AttributeError:
+            raise TypeError(f"Expected File object, got {value}") from None
```

Comment:
if the input key is slice, the value should be `Iterable[FileBase]`, shall we use different error message.